src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/index.tsx

```diff
@@ -444,7 +444,7 @@ const StyledSearchBar = styled(SearchBar)`
     border-bottom-left-radius: 0;
   }
   .search-clear-form,
-  .icon-search {
+  .search-input-icon {
```

Comment:
wouldn't it be better if we could pass a classname for the icon? like` iconClassName?: string `

Reply:
It would but I wanted to scope my changes to just the icons right now as I'm in the middle of slowly replacing all the icons in the app.